methods_and_flow_control/string_practice.rb

```diff
@@ -0,0 +1,11 @@
+#!/usr/bin/env ruby
+
+string = 'this is a string to practice with'
+
+puts string
+puts string.capitalize
+puts string.upcase
+puts string.gsub('string', "'string'").capitalize
+puts "The string '#{string}' has #{string.length} characters."
+puts string.reverse
+puts string.replace('practice practice practice!')
```

Comment:
Seems very similar to `puts 'practice practice practice!`.  Can you find a method on `String` that will return a portion of the string?